internal/pkg/ecs/ecs.go

```diff
@@ -133,6 +136,34 @@ func (c Client) Service(app, env, svc string) (*ecs.Service, error) {
 	return service, nil
 }
 
+// ServiceConnectServices returns a list of services that are in the same
+// service connect namespace as the given service.
+func (c Client) ServiceConnectServices(app, env, svc string) ([]*ecs.Service, error) {
+	s, err := c.Service(app, env, svc)
+	if err != nil {
+		return nil, fmt.Errorf("get service: %w", err)
+	}
+	if len(s.Deployments) == 0 {
+		return nil, nil
+	}
+
+	arns, err := c.ecsClient.ListServicesByNamespace(aws.StringValue(s.Deployments[0].ServiceConnectConfiguration.Namespace))
+	if err != nil {
+		return nil, fmt.Errorf("get services in same namespace: %w", err)
```

Comment:
my language intuition says that we need a "the" before "same" but I might be wrong! :homer-disappearing :elmo_fire
```suggestion
		return nil, fmt.Errorf("get services in the same namespace: %w", err)
```